brave/browser-laptop
# Adds logic for 90% contribution amount

Resolves #14319

## Submitter Checklist:


## Test Plan:
- enable payments
- add 90% - 99% BAT of your monthly budget to your wallet
- make sure that you don't see insufficient notification
- make contribution
- when contribution is done click on payment history icon
- make sure that you see correct amount on the history list (not monthly budget, but actual BAT)
- click on the PDF
- make sure that total is again actual BAT
- sum BAT amounts in the table and make sure that total is correct

## Reviewer Checklist:


Tests
